label_sync/labels.yaml

```diff
@@ -26,6 +26,7 @@ default:
       name: cherrypick-candidate # TODO: rename as cherry-pick-candidate
       target: prs
       addedBy: humans
+      deleteAfter: 2018-09-19T00:00:00Z
```

Comment:
Do we want this to be in the future? :)

Reply:
Thus far I've been treating lazy consensus as: I merge the PR once the deadline passes.  

If I set the deadline in the future, then I'd rather we merge this now, and then lazy consensus objections lead to a revert (or followon PR) rather than update in place.

WDYT @cblecker ?